performance/index.html

```diff
@@ -40,6 +40,14 @@
 <script src="jsonParsers/nearley/nearley_json_parser_tokenizer.js"></script>
 <script src="jsonParsers/nearley/api.js"></script>
 
+<script src="lib/jsonify/jsonify.js"></script>
```

Comment:
Yeah. I searched around for a CDN with it available, but, alas, I didn't find one. So, I threw it in there. :-/